medmodels/treatment_effect/matching/matching.py

```diff
@@ -1,7 +1,14 @@
+"""Matching abstract class.
```

Comment:
this is not the matching abstract class. it is the module that includes the abstract class